README.md

```diff
@@ -229,7 +228,18 @@ const myTemplateString = `
 // editorconfig-checker-enable
 ```
 
-### Excluding Files
+### Excluding Paths
+
+You can exclude paths from being checked in several ways:
+- ignoring a file by documenting it inside the to-be-excluded file
+- adding a regex matching the path to the `.ecrc`
+- passing a regex matching the path as argument to `--exclude`
+
+All these excludes are used in addition to the [default excludes](#default-excludes), unless you opt out of them.
```

Comment:
To make it clearer. :smile:

```suggestion
All these excludes are used in addition to the [default excludes](#default-excludes), unless you opt out of them with the `--ignore-defaults` flag.
```

Reply:
I fixed this a slightly different way: I created an entire new section describing the `IgnoreDefaults` option and command line switch, linking to it from all the places it is mentioned.